integration_tests/test_asset_recording.py

```diff
@@ -55,6 +55,20 @@ def test_prime_records_snapcraft_yaml_copy(self):
 
 class ManifestRecordingTestCase(AssetRecordingBaseTestCase):
 
+    def test_prime_records_uname(self):
+        self.run_snapcraft('prime', project_dir='basic')
+
+        recorded_yaml_path = os.path.join(
+            self.prime_dir, 'snap', 'manifest.yaml')
+        with open(recorded_yaml_path) as recorded_yaml_file:
+            recorded_yaml = yaml.load(recorded_yaml_file)
+
+        expected_uname = subprocess.check_output(
+            ['uname', '-a']).decode('utf-8').strip()
```

Comment:
We've started a pattern of using `.decode(sys.getfilesystemencoding())` for this type of thing. Should we be using that here as well?